engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceMigrationMigrateProcessor.java

```diff
@@ -346,4 +369,27 @@ private static final class ElementWithIncidentException extends RuntimeException
               processInstanceKey, elementId));
     }
   }
+
+  /**
+   * Exception that can be thrown during the migration of a process instance, in following cases:
+   *
+   * <p>1) A mapping instruction contains a source element id that does not exist in the source
+   * process definition.
+   *
+   * <p>2) A mapping instruction contains a target element id that does not exist in the target
+   * process definition.
```

Comment:
:upside_down_face: You can use HTML tags to create an ordered list here (some examples are documented [here](https://blog.joda.org/2012/11/javadoc-coding-standards.html)).

Reply:
Good one! 🚀 Updated!